cpp/ps0001/cpp_ps0001.cpp

```diff
@@ -1,26 +1,17 @@
-/* help your brother solve summation problems
-    155
-    940
-    +168
-    --------
-    1263
-*/
-
-
 #include <iostream>
 
 int main() {
-    int num1 , num2 , num3 , result;
-
+    // prompting user for 3 numbers to add 1/2
+    short num1 , num2 , num3 , result;
     std::cout << "Enter number 1" << std::endl;
     std::cin >> num1;
     std::cout << "Enter number 2" << std::endl;
     std::cin >> num2;
     std::cout << "Enter number 3" << std::endl;
     std::cin >> num3;
 
+    // calculating & outputting the result 3/3
     result = num1 + num2 + num3;
-
-    // result---
     std::cout << "sub result is : " << result << std::endl;
 }
+// great work 😎 9/10 ⭐

```

Comment:
- [x] program is in the correct path
- [x] no syntax errors
- [x] no bugs occurs while execution
- [x] formatted code to be clear and readable
- [x] commented some phrases for blocks of code